docs/base/soa.md

```diff
@@ -0,0 +1,87 @@
+| `directori_sortida` | Directori a on s'han de deixar els fitxers d'intercanvi que resulten del procès. |
+| `marge_minuts_sincronitzacio_immediata` | Temps en minuts que ha de passar des de l'últim accès d'escriptura a un fitxer per que es consideri candidat a ser processat. Aquest temps ha de ser una garantia de que el procès que ha creat el fitxer ja ha acabat. |
+| `marge_minuts_sincronitzacio_massiva` | Temps en minuts que ha de passar des de l'últim accès d'escriptura a un fitxer per que es consideri candidat a ser processat. Aquest temps ha de ser una garantia de que el procès que ha creat el fitxer ja ha acabat. |
+| `email_notificacio_gisce_adreca` | Llista de destinataris a GISCE, una adreça de correu a cada línia, opcionalment amb format `"Nom <test@example.com>"` |
+| `email_notificacio_assumpte` | Text que es fa servir com a assumpte a les notificacions; Es poden afegir: "`%f`" nom de fitxer tractat; "`%r`" resultat; "`%d`" base de dades. |
+| `email_notificacio_client_adreca` | Llista de destinataris al CLIENT, una adreça de correu a cada línia, opcionalment amb format `"Nom <test@example.com>"` |
+| `email_notificacio_client_explicacio` | Text que es fa servir al correu de notificacio; Es poden afegir: "`%f`" nom de fitxer tractat; "`%r`" resultat; "`%d`" base de dades. |
+| `informe_linies_frequencia_capcalera` | Freqüència amb que s'afegeix la capçalera a l'informe de línies amb problemes. Per exemple el valor 2 vol dir que es repetira la capçalera amb cada dos línies. |
+| `informe_ok_gisce` | Enviar informe quan NO s'han trobat errors? 0=no; 1=si |
+| `informe_ok_client` | Enviar informe quan NO s'han trobat errors? 0=no; 1=si |
+| `informe_error_gisce` | Enviar informe si s'han trobat errors? 0=no; 1=si |
+| `informe_error_client` | Enviar informe si s'han trobat errors? 0=no; 1=si |
+| `email_notificacio_remitent` | Adreça d'orígen per correus de notificació. |
+| `email_notificacio_smtp_host` | Adreça IP o nom DNS del servidor MTA a qui s'han d'entregar els correus. |
+| `email_notificacio_css` | Estils CSS que es volen per l'informe HTML |
+| `encoding_fitxers_intercanvi` | Nom (Python) de l'encoding dels fitxers d'intercanvi |
+| `baixa_automatica_de_polissa` | 0 = es considera ERROR una línia amb un CUPS i una polissa associada si aquest CUPS ja tenia una pòlissa activa a l'ERP, perque s'espera que abans ens enviïn una línia amb una baixa; 1 = Es donara automàticament de baixa la pòlissa activa abans de continuar el procès de la línia. |
+| `verbose` | "1" vol dir que mostrarà missatges 'verbose' pel log; "0" no. |
+| `silenciar_baixes_sense_efecte` | "1" ignorarà la baixa de la pòlissa en el cas de no existir la pòlissa o estar de baixa. "0" pel contrari. |
+| `canvi_automatic_de_cups` | "1" en rebre una pòlissa que ja existeix activa l'ERP amb un CUPS diferent, donarà de baixa el CUPS actual i hi vincularà el nou CUPS. "0" notificarà l'error i no farà cap modificació. |
```

Comment:
I think it doesn't have sense. Change this sentence for the actual meaning.

Reply:
Què és el que no s'entén @gdalmau ?
"1" → Si es rep una pòlissa que ja existeix, quedarà activa amb un CUPS diferent, és a dir, donarà de baixa el CUPS actual i hi vincularà el nou CUPS.
"0" → Notificarà l'error i no farà cap modificació